extensions/hibernate-search-orm-elasticsearch/deployment/src/main/resources/dev-ui/hibernate-search-+-elasticsearch/hibernate-search-orm-elasticsearch-indexed-entity-types.js

```diff
@@ -0,0 +1,136 @@
+import { LitElement, html, css} from 'lit';
```

Comment:
Notice the weird directory name:

```
extensions/hibernate-search-orm-elasticsearch/deployment/src/main/resources/dev-ui/hibernate-search-+-elasticsearch/hibernate-search-orm-elasticsearch-indexed-entity-types.js
```

That's because the name of the extension is `Hibernate Search + Elasticsearch`...

Shouldn't we use artifact IDs instead of extension names for such things?

Reply:
Right, that's the kind of things that worries me. But currently I have no choice :/

I could change the extension name, but it seems weird to me that it's used for anything else that just display... ?

Reply:
We have discussed that, I'll soon make a change to use artifact name rather than name, and as part of that change I'll do the conversion to that new format in all extensions that already use this flawed name, so for this PR we can just ignore the windows failure (if any)

Reply:
@yrodiere - this is now fix with Auto naming here: https://github.com/quarkusio/quarkus/pull/32101

Reply:
That's great, much simpler. Thanks @phillip-kruger!